mscore/shortcut.cpp

```diff
@@ -1934,6 +1934,13 @@ Shortcut Shortcut::_sc[] = {
          QT_TRANSLATE_NOOP("action","Voice 4"),
          0,
          },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "voice-selection-cycle",
+         QT_TRANSLATE_NOOP("action","Cycle through voices in range-selection (ChordRests only)"),
```

Comment:
"(ChordRest only)" is too technical, "(Chords and rests only)" woule be better IHMO